dokka-subprojects/plugin-base/src/main/resources/dokka/styles/style.css

```diff
@@ -1023,6 +1024,7 @@ h1.cover {
     font-size: inherit;
     line-height: inherit;
     transition: color .1s, border-color .1s;
+    word-wrap: break-word;
```

Comment:
nit: probably it'd be more consistent to rename this to `overflow-wrap` since it is a more common synonym